packages/vite/src/node/plugins/asset.ts

```diff
@@ -325,6 +325,12 @@ function isGitLfsPlaceholder(content: Buffer): boolean {
   // Check whether the content begins with the characteristic string of Git LFS placeholders
   return GIT_LFS_PREFIX.compare(content, 0, GIT_LFS_PREFIX.length) === 0
 }
+// spaces in not supported in srcset
+const svgDataURISupportedHexCode: Record<string, string> = {
+  '%3D': '=',
+  '%3A': ':',
+  '%2F': '/',
+}
```

Comment:
Would be good to move this closer to the extract function too

Reply:
We use this iconify `encodeSVGforURL` function to encode svg in UnoCSS: https://github.com/iconify/iconify/blob/main/packages/utils/src/svg/url.ts